master/getting-started/kubernetes/installation/gce.md

```diff
@@ -2,50 +2,22 @@
 title: Deploying Calico and Kubernetes on GCE
 ---
 
-These instructions allow you to set up a Kubernetes cluster with Calico networking on GCE using the [Calico CNI plugin][cni-plugin]. This guide does not setup TLS between Kubernetes components or on the Kubernetes API.
+GCE provides a platform that can be used to
+[deploy a Kubernetes cluster][running-GCE-on-K8s] with minimal customization
+when following one of the many possibilities for deploying Kubernetes and
+then [installing Calico](.).  The information here provides the customization needed.
 
-## 1. Getting started with GCE
+### Setting up GCE networking
 
-These instructions describe how to set up two CoreOS Container Linux hosts on GCE.  For more general background, see
-[the CoreOS on GCE documentation][coreos-gce].
-
-### 1.1 Install the gcloud tool
-
-If you already have the `gcloud` utility installed, and a GCE project configured, you may skip this step.
-
-Download and install GCE, then restart your terminal:
-
-```shell
-curl https://sdk.cloud.google.com | bash
-```
-
-For more information, see Google's [gcloud install instructions][gcloud-instructions].
-
-Log into your account:
-
-```shell
-gcloud auth login
-```
-
-In the GCE web console, create a project and enable the Compute Engine API.
-Set the project as the default for gcloud:
-
-```shell
-gcloud config set project PROJECT_ID
-```
-
-And set a default zone
-
-```shell
-gcloud config set compute/zone us-central1-a
-```
-
-### 1.2 Setting up GCE networking
-
-GCE blocks traffic between hosts by default; run the following command to allow Calico traffic to flow between
+In addition to allowing all traffic between the hosts for the Kubernetes
```

Comment:
This statement makes it sound like you've already explained the need to "allow all traffic between the hosts".

Furthermore, is that really a requirement? What are the bare minimum connections calico needs on the host? We don't necessarily have to figure this out and document it now but we should silently leave it out if that's the case.